fischkapp/src/components/AddNewCard/AddNewBackCard.tsx

```diff
@@ -1,28 +1,62 @@
+import { flashCardsData } from "../../data/data";
 import { StyledCardContent } from "../styles/CardGlobal.styled";
 import { StyledInput } from "../styles/AddNewCard.styled";
 import { StyledCardButton } from "../styles/buttons/CardButton";
 import { StyledButtonsContainer } from "../styles/AddNewCard.styled";
 import { StyledControlCardPanel } from "../styles/AddNewCard.styled";
 import { StyledDeleteButton } from "../styles/buttons/DeleteButton";
 import deleteIcon from "../../assets/deleteIcon.svg";
+import { useState } from "react";
 
 type BackCardProps = {
   onFlipCard: Function;
+  frontText: String;
 };
 
-export const AddNewBackCard = ({ onFlipCard }: BackCardProps) => {
+interface FlashCardType {
+  id?: number;
+  frontText: String;
+  backText: String;
+}
+
+export const AddNewBackCard = ({ onFlipCard, frontText }: BackCardProps) => {
+  const [backText, setBackText] = useState("");
+  const [flashCards, setFlashCards] = useState<FlashCardType[]>(flashCardsData);
```

Comment:
This should be in `App.tsx`, so you can manage added cards to other parts of your app